src/NearOrg/Notifications/Notification.jsx

```diff
@@ -124,14 +124,48 @@ const Left = styled.div`
 const Right = styled.div``;
 
 const { value } = props;
-const { type } = value;
+let { type } = value;
 const item = value?.item || {};
 const path = item.path || "";
 
 // Build notification
 let { blockHeight, accountId } = props;
 let postUrl = "";
 
+// Construct DevGov postUrl
+function buildPostUrl(widgetName, linkProps) {
+  linkProps = { ...linkProps };
+
+  const nearDevGovGigsWidgetsAccountId =
+      props.nearDevGovGigsWidgetsAccountId ||
+      (context.widgetSrc ?? "devgovgigs.near").split("/", 1)[0];
+
+  if (props.nearDevGovGigsContractAccountId) {
+    linkProps.nearDevGovGigsContractAccountId =
+        props.nearDevGovGigsContractAccountId;
+  }
+
+  if (props.nearDevGovGigsWidgetsAccountId) {
+    linkProps.nearDevGovGigsWidgetsAccountId =
+        props.nearDevGovGigsWidgetsAccountId;
+  }
+
+  if (props.referral) {
+    linkProps.referral = props.referral;
+  }
+
+  const linkPropsQuery = Object.entries(linkProps)
+      .filter(([_key, nullable]) => (nullable ?? null) !== null)
+      .map(([key, value]) => `${key}=${value}`)
+      .join("&");
+
+  return `/${nearDevGovGigsWidgetsAccountId}/widget/gigs-board.pages.${widgetName}${
+      linkPropsQuery ? "?" : ""
+  }${linkPropsQuery}`;
+}
+
+console.log({props})
```

Comment:
Remove it please

Reply:
Sure! Sorry about that, didn't see it 😀

Reply:
Your code looks good! Just need to check that locally. May I ask you to add a couple of screenshots here?